rust/perspective-python/Cargo.toml

```diff
@@ -10,45 +10,51 @@
 #  ┃ of the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). ┃
 #  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 
-from enum import Enum
+[package]
```

Comment:
This should probably remain in the `python` folder as it is still the python codebase, it can be linked to the rust code straightforwardly with a cargo workspace

Reply:
There is no `python` folder on this branch.

Reply:
thats fine then, just an organizational choice as `perspective-python` is not a useable standalone rust package. But its convenient to keep things together, and `packages` is already taken.